src/subscription_manager/cli_command/environments.py

```diff
@@ -74,7 +74,7 @@ def _do_command(self):
             if self.options.token:
                 self.cp = self.cp_provider.get_keycloak_auth_cp(self.options.token)
             else:
-                if not self.options.enabled:
+                if not self.options.enabled and (self.options.username is None or self.options.password is None):
```

Comment:
This change completely break using CLI options as you can see:

```
[root@thinkpad-t580 subscription-manager]# PYTHONPATH=./src python -m \
    subscription_manager.scripts.subscription_manager \
    environments --username admin --password admin --org admin
Invalid credentials. (HTTP error code 401: Unauthorized)

[root@thinkpad-t580 subscription-manager]# PYTHONPATH=./src python -m \
    subscription_manager.scripts.subscription_manager \
    environments --username admin --password admin --org snowwhite
Invalid credentials. (HTTP error code 401: Unauthorized)

[root@thinkpad-t580 subscription-manager]# PYTHONPATH=./src python -m \
    subscription_manager.scripts.subscription_manager environments
This operation requires user crendentials
Username: admin
Password: 
Hint: User "admin" is member of following organizations: admin, snowwhite, donaldduck
Organization: admin
This list operation does not have any environments to report.

[root@thinkpad-t580 subscription-manager]# PYTHONPATH=./src python -m \
    subscription_manager.scripts.subscription_manager environments
This operation requires user crendentials
Username: admin
Password: 
Hint: User "admin" is member of following organizations: admin, snowwhite, donaldduck
Organization: snowwhite
+-------------------------------------------+
          Environments
+-------------------------------------------+
Name:        env-name-1
Description: Testing environment #1

Name:        env-name-2
Description: Testing environment #2
```

And I can still see typo "crendentials"